charts/gatekeeper/templates/gatekeeper-validating-webhook-configuration-validatingwebhookconfiguration.yaml

```diff
@@ -61,6 +61,9 @@ webhooks:
     {{- if .Values.enableDeleteOperations }}
     - DELETE
     {{- end }}
+    {{- if .Values.enableConnectOperations }}
```

Comment:
@thomaschaplin can you revert changes in `/charts`? `manifest_staging/charts` is automatically promoted to `/charts` when we cut a release

Reply:
@sozercan I've done that now, thanks for confirming - I wasn't sure as the docs wasn't clear to me if this was auto-generated or not.